components/ui/accordion.tsx

```diff
@@ -0,0 +1,151 @@
+        </AccordionTrigger>
+        <AccordionContent>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Blanditiis
+          vitae unde at ratione laboriosam praesentium non impedit eos accusamus
+          libero debitis tempore, voluptatum ipsam, molestiae perferendis, nisi
+          distinctio temporibus corporis?
+        </AccordionContent>
+      </AccordionItem>
+      <AccordionItem value={'what-documents-are-required'}>
+        <AccordionTrigger>
+          Які документи потрібні для отримання стипендії?
+        </AccordionTrigger>
+        <AccordionContent>
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Maxime
+          dolorem quae nulla reiciendis, est dolorum recusandae omnis quam
+          necessitatibus totam. Quis mollitia perspiciatis incidunt? Fuga eius
+          soluta repudiandae excepturi tempore!
+        </AccordionContent>
+      </AccordionItem>
+    </Accordion>
+  );
+};
+
+const AccordionItem = React.forwardRef<
+  React.ElementRef<typeof AccordionPrimitive.Item>,
+  React.ComponentPropsWithoutRef<typeof AccordionPrimitive.Item>
+>(({ className, ...props }, ref) => (
+  <AccordionPrimitive.Item
+    ref={ref}
+    className={cn('border rounded-s-xl rounded-e-xl font-sans', className)}
+    {...props}
+  />
+));
+
+AccordionItem.displayName = 'AccordionItem';
+
+const AccordionTrigger = React.forwardRef<
+  React.ElementRef<typeof AccordionPrimitive.Trigger>,
+  React.ComponentPropsWithoutRef<typeof AccordionPrimitive.Trigger>
+>(({ className, children, ...props }, ref) => {
+  const [isCollapsed, setIsCollapsed] = React.useState(true);
+
+  return (
+    <AccordionPrimitive.Header className="flex">
+      <AccordionPrimitive.Trigger
+        onClick={() => setIsCollapsed(!isCollapsed)}
+        ref={ref}
+        className={cn(
+          'flex flex-1 lg:max-w-screen-2xl md:max-w-screen-md sm:max-w-xs select-none justify-between font-medium hover:underline [&[data-state=open]>svg]:rotate-180 lg:text-lg md:text-xl sm:text-2xl p-2 text-center',
+          className,
+        )}
+        {...props}
+      >
+        {children}
+        {isCollapsed ? <PlusIcon /> : <Minus />}
+      </AccordionPrimitive.Trigger>
+    </AccordionPrimitive.Header>
+  );
+});
+
+AccordionTrigger.displayName = AccordionPrimitive.Trigger.displayName;
+
+const AccordionContent = React.forwardRef<
+  React.ElementRef<typeof AccordionPrimitive.Content>,
+  React.ComponentPropsWithoutRef<typeof AccordionPrimitive.Content>
+>(({ className, children, ...props }, ref) => {
+  return (
+    <AccordionPrimitive.Content
+      ref={ref}
+      className="data-[state=closed]:animate-accordion-up data-[state=open]:animate-accordion-down"
+      {...props}
+    >
+      <div className={cn('p-2', className)}>{children}</div>
+    </AccordionPrimitive.Content>
+  );
+});
+
+AccordionContent.displayName = AccordionPrimitive.Content.displayName;
+
+export default FAQAccordion;
```

Comment:
According to documentation, the component should export `{ Accordion, AccordionItem, AccordionTrigger, AccordionContent }`, so the developer can use accordion as he wants. 
Remove `FAQAccordion` and style accordion items respectively to match the design